app/helpers/stacks_helper.rb

```diff
@@ -1,4 +1,16 @@
 module StacksHelper
+  DISABLED_BUTTON = %w(btn--disabled disabled).freeze
```

Comment:
Is this used anywhere? You're using `classes = %W(btn btn--primary deploy-action #{commit.state})` on line `6`.